.github/workflows/build.yml

```diff
@@ -73,10 +78,23 @@ jobs:
             !bin/**/*.exp
           retention-days: 30
 
+      - name: Upload artifacts (macOS)
+        if: matrix.os == 'macos-latest'
+        uses: actions/upload-artifact@v3
+        with:
+          name: godot-python-macos-universal
+          path: |
+            bin/**/*
```

Comment:
Are these lines needed for macos?

Reply:
I suppose not... 

I failed to check if those files were generated in the first place, though the python folder also looks suspiciously empty so that's probably the next thing I will have to take a look at.

![SCR-20241025-calj](https://github.com/user-attachments/assets/7422eea0-3776-475a-a3ef-58c3e4060823)